api/.pylintrc

```diff
@@ -560,7 +562,7 @@ max-bool-expr=5
 max-branches=12
 
 # Maximum number of locals for function / method body.
-max-locals=15
+max-locals=20
```

Comment:
Similar to the above, extracting functions/methods reduces the amount of locals: https://refactoring.com/catalog/extractFunction.html
(If there's consensus on this, let's not address it in this PR, wait for refactor task)

Reply:
I've added this comment to the tech debt ticket.